R/release.r

```diff
@@ -294,25 +294,30 @@ build_cran <- function(pkg, args) {
   built_path
 }
 
+upload_to <- function(dest, pkg, built_path) {
```

Comment:
As the `dest` is no longer just the known hard-coded CRAN URL, perhaps should include a cleaner on the parameter. In `remotes` pkg, they pass a [trimmed whitespace URL](https://github.com/r-lib/remotes/blob/9b5f7a73021999b7317507449336ea8a5e9fbdfc/R/install-remote.R#L197) to their remote functions. Could similarly here call the `trim_ws()` function.